pkg/agent/qrm-plugins/memory/dynamicpolicy/policy.go

```diff
@@ -258,6 +260,10 @@ func (p *DynamicPolicy) Start() (err error) {
 
 	go wait.BackoffUntil(communicateWithMemoryAdvisorServer, wait.NewExponentialBackoffManager(800*time.Millisecond,
 		30*time.Second, 2*time.Minute, 2.0, 0, &clock.RealClock{}), true, p.stopCh)
+
+	p.lwRecvTimeMonitor = timemonitor.NewTimeMonitor(memoryAdvisorLWRecvTimeMonitorName,
+		memoryAdvisorLWRecvTimeMonitorDurationThreshold, util.MetricNameLWRecvStuck, p.emitter)
+	go wait.Until(p.lwRecvTimeMonitor.MonitorRefreshTime, memoryAdvisorLWRecvTimeMonitorPeriod, p.stopCh)
```

Comment:
Why doesn't the Time Monitor expose the `Run(<-ch)` method instead of `MonitorRefreshTime`? The `Run(<-ch)` method is more general and flexible.